### PR TITLE
CODE RUB: Change Model Prompt Property to Prompts

### PR DESCRIPTION
**Pull request by Ulugbek4499**
closes #31

### CODE REVIEW

File: OpenAI.NET/Models/ExternalCompletions/ExternalCompletionRequest.cs [hunk 1]
```diff
@@ -12,8 +12,8 @@ internal class ExternalCompletionRequest
         [JsonProperty(propertyName: "model")]
         public string Model { get; set; }
 
-        [JsonProperty(propertyName: "prompt")]
-        public string[] Prompt { get; set; } = new string[0];
+        [JsonProperty(propertyName: "prompts")]
```
Comment (hassanhabib) [inline OpenAI.NET/Models/ExternalCompletions/ExternalCompletionRequest.cs:15]: The API accepts only `prompt` but C# should say `Prompts`

Comment (Ulugbek4499) [reply OpenAI.NET/Models/ExternalCompletions/ExternalCompletionRequest.cs]: Resolved

**Comment by hassanhabib** [comment]
@Ulugbek4499 please fix conflicts

**Comment by Ulugbek4499** [comment]
@hassanhabib 
Sorry I didn't see any conflict, could you explain me that, please!

File: OpenAI.NET/Models/ExternalCompletions/ExternalCompletionRequest.cs (hunk 1 above)
Comment (hassanhabib) [reply OpenAI.NET/Models/ExternalCompletions/ExternalCompletionRequest.cs]: This is still not resolved.
We need to change `JsonProperty(propertyName: "prompts")` to `JsonProperty(propertyName: "prompt")`

**Comment by hassanhabib** [comment]
@ElbekDeveloper can you help @Ulugbek4499 with conflicts resolutions?

**Comment by hassanhabib** [comment]
Closing this due to inactivity

**Pull request closed without merging**